src/index.html

```diff
@@ -2,16 +2,122 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form>
+      <fieldset>
+        <legend>Personal information</legend>
+        <label>
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          required autocomplete="off">
+        </label>
+        <label class="form_field">
+          Name:
+          <input
+          type="text"
+          name="name"
+          required autocomplete="off">
+        </label>
+        <label class="form_field">How old are You?
+          <input
+          type="number"
+          name="age"
+          min="1"
+          max="100"
+          value="12">
+        </label>
+        <label class="form_field" for="date_of_birth">Full date of birth:
+          <input type="date" name="dateOfBirth">
+        </label>
+        <label class="form_field">I accept the term of the agreement
+          <input type="checkbox">
+        </label>
+      </fieldset>
+      <fieldset class="field">
+        <legend>Registration</legend>
+        <label>
+          E-mail:
+          <input
+          type="email"
+          name="email"
+          placeholder="email@example.com">
+        </label>
+        <label class="form_field">
+          Password:
+          <input
+          type="password"
+          name="password"
+          minlength="5"
+          maxlength="16"
+          required>
+        </label>
+      </fieldset>
+      <fieldset class="field">
+        <legend>An interesting fact about you!</legend>
+        <label>
+          Do you love cats?
+          <input
+          type="radio"
+          name="cat"
+          value="yes">  Yes
```

Comment:
```suggestion
          value="yes"
         >
         Yes
```

text should start from new line
and check it below with `No`